src/var/mash.jl

```diff
@@ -0,0 +1,34 @@
+# MinHash
+# ========
+#
+# Functions to MASH distance for MinHash sketches
+#
+# see DOI: 10.1186/s13059-016-0997-x
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+function jaccarddistance(sketch1::MinHashSketch, sketch2::MinHashSketch)
+    d = length(setdiff(sketch1.sketch, sketch2.sketch))
```

Comment:
@bicycle1885 I initially wrote something out to do something like what you're suggesting (you can look [here](https://github.com/kescobo/MashTun/blob/ab044da78095595f8a3b6379cd49903f48ba8d1f/src/MashTun.jl) starting at ln 117 for what I tried), but it didn't seem to be any faster. Probably my lack of knowledge for how to do this efficiently is to blame - happy to take guidance on that.

@Ward9250 the calculation is the length of the intersect over the length of the union. `setdiff` tells me how many items are in one but not the other, and since we know the size of each set, that's enough information to know how many are shared as well (we don't need to know what's shared, just the number).